### PR TITLE
Add home page

### LINKED ISSUE #30: frontEnd : add home page

### PR DESCRIPTION
**Pull request by mehmetfatiherdem**
closes #30

Screenshot from 2022-02-23 18-37-40
Screenshot from 2022-02-23 18-38-18
Screenshot from 2022-02-23 18-40-43
Screenshot from 2022-02-23 18-41-05

Screenshot from 2022-02-23 18-40-52
Screenshot from 2022-02-23 18-41-14
Screenshot from 2022-02-23 18-41-21

### CODE REVIEW

**Comment by Shrreya** [comment]
@Peri7at Can you review this PR today? I'll add my review after yours.

**Comment by Shrreya** [comment]
@mehmetfatiherdem Can you add a screenshot of the home page in the PR description?

**Comment by Peri7at** [comment]
Great job @mehmetfatiherdem.
It looks good overall though in my PC, homepage looks like this.. I don't know if it is in our scope to arrange homepage according to screen sizes..
image
image
5DC0429F-27B0-45A4-9DBE-A2BBDE9D96F8_1_102_o

**Review by Peri7at — COMMENTED** [review]
Can you make same right and left margins for all sections?

**Comment by Peri7at** [comment]
I suggest to change logo for the following photo, what do you think?
people-circle

**Comment by AmmarBaki2** [comment]
> I suggest to change logo for the following photo, what do you think? people-circle

It looks great 👍

**Comment by mehmetfatiherdem** [comment]
> I suggest to change logo for the following photo, what do you think? people-circle

sure we can use it :+1:

**Comment by mehmetfatiherdem** [comment]
> Great job @mehmetfatiherdem. It looks good overall though in my PC, homepage looks like this.. I don't know if it is in our scope to arrange homepage according to screen sizes.. image image image

Yes, I think it is not that responsive :sweat_smile:.

**Comment by mehmetfatiherdem** [comment]
> Can you make same right and left margins for all sections?

I can try that :+1:

**Review by Peri7at — APPROVED** [review]

File: src/public/css/home.css [hunk 1]
```diff
@@ -62,7 +62,6 @@ ul {
   display: flex;
   justify-content: space-around;
   align-items: center;
-  flex-wrap: wrap;
   margin-top: 50px;
 }
```
Comment (Peri7at) [inline src/public/css/home.css:66]: Could you add  the same right and left margins to overview also. All sections look aligned now beside overview section.

**Comment by mehmetfatiherdem** [comment]
Could you review this PR @Shrreya?

**Review by Shrreya — CHANGES_REQUESTED** [review]
@mehmetfatiherdem I'm noting down all changes here below.

1. Why is there a screenshot of the API docs? That's not required, once the swagger docs are merged you can just link the navigation item "API Documentation" to the swagger docs route.
2. The screenshots for designs are not looking good. Can you use the Figma embed links instead? Then users can actually move around and interact with the design diagrams?
3. Can you mark my comments on Figma as resolved? Then my "S" bubbles won't show on your designs.
4. Remove the sign in and sign up buttons from the navigation header. Users will authenticate from the swagger docs.
5. Let's use a different color other than black for titles like "Overview", "Items", "Requests" and so on. I think the green color of the header bar can be used.
6. Reduce the font-weight of the paragraph texts. Right now everything seems to be in bold text.
7. I think you can mention a few more technologies like ExpressJS, ESLint, Prettier, Husky, Jest, AWS, Nginx. Be creative and think about what other cool technologies were used.
8. You can show team members in 2 rows, 3 on each row for better spacing.
9. Mention roles of each member below their name. (Lead Developer, Backend Developer, Code Reviewer)
10. Remove twitter, google and facebook buttons.
11. Make sure footer items are linked to correct sections of the page.
12. Replace "Company name" in footer with "Paying it forward | Re:Coded".

**Review by Shrreya — APPROVED** [review]

**Pull request merged**